src/sys/socket/mod.rs

```diff
@@ -562,6 +562,11 @@ pub enum ControlMessageOwned {
     /// [Further reading](https://www.kernel.org/doc/html/latest/networking/timestamping.html)
     #[cfg(all(target_os = "linux"))]
     ScmTimestampns(TimeSpec),
+    /// Configurable nanoseconds resolution timestamps
```

Comment:
Could you please explain what the wrapped type is?  Why are there three timespecs?

Reply:
https://www.kernel.org/doc/html/latest/networking/timestamping.html#scm-timestamping-records

Reply:
A link to an external website is not sufficient documentation.  Why is the wrapped value an array?  What's special about "3"?

Reply:
The link was specifically for your question. Have you checked it out?

Reply:
No; I'm asking for Nix's own documentation to describe why the value is a 3-member array.  You can leave the external link in there, similar to how many Nix functions have man page links, but Nix's own documentation should have at least basic information about what the type is.

Reply:
Ok. Added some documentation.